packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -40,14 +47,17 @@ export const getStyles = memoizeFunction(
 
               selectors: {
                 [HighContrastSelector]: {
-                  color: 'Window',
-                  backgroundColor: 'WindowText',
+                  color: 'WindowText',
+                  backgroundColor: 'Window',
+                  border: '1px solid WindowText',
+                  borderRightWidth: '0',
                   MsHighContrastAdjust: 'none',
                 },
+                ...getEdgeChromiumNoHighContrastAdjustSelector(),
```

Comment:
i dont think this is needed, can you remove and check?

Reply:
not needed, removed